src/Sarif/Autogenerated/Address.cs

```diff
@@ -42,7 +43,7 @@ public SarifNodeKind SarifNodeKind
         [DataMember(Name = "absoluteAddress", IsRequired = false, EmitDefaultValue = false)]
         [DefaultValue(-1)]
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
-        public int AbsoluteAddress { get; set; }
+        public BigInteger AbsoluteAddress { get; set; }
```

Comment:
>BigInteger [](http://example.com/codeflow?start=15&length=10)

After this is merged there will be follow up PR(s) to apply the same change to else appropriate.